tests/test_incident.py

```diff
@@ -101,3 +101,9 @@ def test_inc_nochannels2(mock_good):
     ]
     with pytest.raises(EmptyChannels):
         Incident(bad_data)
+
+
+def test_inc_revisions(mock_good):
+    incident = Incident(test_data)
+    assert incident.revisions_with_fallback("x86_64", "15-SP4")
+    assert incident.revisions_with_fallback("aarch64", "15-SP4")
```

Comment:
I can't figure out how to reproduce the case where we get a SLE12 with no versioning...

Reply:
Works now. Typos can be confusing in this context.